askama_shared/src/filters/mod.rs

```diff
@@ -518,21 +518,21 @@ mod tests {
     #[test]
     #[allow(clippy::float_cmp)]
     fn test_into_f64() {
-        assert_eq!(into_f64(&1).unwrap(), 1.0 as f64);
-        assert_eq!(into_f64(&1.9).unwrap(), 1.9 as f64);
-        assert_eq!(into_f64(&-1.9).unwrap(), -1.9 as f64);
+        assert_eq!(into_f64(&1).unwrap(), 1.0_f64);
```

Comment:
I know clippy suggested the `_f64` suffix, but I think type suffixes without underscores are more idiomatic. What do you think?

Reply:
If you prefer without, then I can remove the underscores. :)

In this case I actually prefer with. It's less because of those cases, but more in the case of hexadecimals, e.g. `0xabcdi32` vs `0xabcd_i32`. So for consistency, I always use underscore.

In my own projects I even enforce some pedantic lints, so I'm used to underscores in literals given [`unreadable_literal`](https://rust-lang.github.io/rust-clippy/master/index.html#unreadable_literal).